cmd/parca-agent/main.go

```diff
@@ -283,6 +287,12 @@ func main() {
 		os.Exit(1)
 	}
 
+	bpf.SetLoggerCbs(bpf.Callbacks{
```

Comment:
Moved the libbpf logger setter here to ensure that we do this before loading any BPF program